src/components/Swap/Summary/index.tsx

```diff
@@ -27,59 +30,176 @@ const PriceImpactText = styled.span`
   color: ${({ theme }) => theme.error};
 `
 
+const PermitTooltipBody = styled.div`
+  max-width: 220px;
+`
```

Comment:
this is now being re-used in a lot of places, just to constrain the tooltip body. can we make a shared component exported from the tooltip code?

Reply:
Sure

Reply:
This should probably be done is sep refactor PR tho since it touches unrelated files

Reply:
i actually did this in a PR of mine because i had the same need: `SmallToolTipBody`. you could use it here but it's not blocking